### DISCUSSION TITLE
How does the linkerd control plane cancel the ha mode?

### DISCUSSION THREAD

**Discussion by eddielishan** [linkerd/linkerd2#8465] [Q&A] [answered]
Excuse me, how does the linkerd control plane cancel the ha mode without reinstalling it.thanks

**Comment by mateiidavid** [comment]
@eddielishan you might have to edit the PodDisruptionBudget and the replicas in all respective deployments.

**Comment by eddielishan** [comment]
@mateiidavid 

I mainly want to be able to deploy services with the proxy injector is down，but I modify the configuration doesn t work

linkerd-config  ： webhookFailurePolicy: Ignore

The HA proxy injector is deployed with a stricter failure policy to enforce [automatic proxy injection](https://linkerd.io/2.11/features/proxy-injection/). This setup ensures that no annotated workloads are accidentally scheduled to run on your cluster, without the Linkerd proxy. (This can happen when the proxy injector is down.)

If proxy injection process failed due to unrecognized or timeout errors during the admission phase, the workload admission will be rejected by the Kubernetes API server, and the deployment will fail.

**Comment by adleong** [comment, marked as answer]
Hi @eddielishan.  If you're using the Linkerd CLI, and have installed Linkerd in HA mode, there isn't an easy way to change to non-HA mode.  When you install Linkerd with the HA flag, it configures a number of different values which you can see in https://github.com/linkerd/linkerd2/blob/main/charts/linkerd-control-plane/values-ha.yaml

You can also see your full installed configuration by looking at:

```
kubectl get -n linkerd cm/linkerd-config -ojsonpath='{.data.values}'
```

You can change these values by running `linkerd upgrade` with the `--set` flag.  To fully disable HA mode, you can set each of the values back to their non-HA values from https://github.com/linkerd/linkerd2/blob/main/charts/linkerd-control-plane/values.yaml

Of course, if there are only certain values that you care about such as `webhookFailurePolicy` or `controllerReplicas` for example, feel free to just focus on those.

**Comment by eddielishan** [comment]
Thank you for your patience. The question has been solved
